server/manifestmergetool/src/main/java/com/defold/manifestmergetool/MergePolicy.java

```diff
@@ -0,0 +1,18 @@
+package com.defold.manifestmergetool;
+
+public enum MergePolicy {
+    KEEP,
+    MERGE,
+    REPLACE;
```

Comment:
I added REPLACE as a new policy. It is not really needed for this PR but it makes sense to have it.